### PR TITLE
mat: export Dense.solveTwoChol

### PR DESCRIPTION
**Pull request by kortschak**
@btracey Please take a look.

### CODE REVIEW

**Comment by btracey** [comment]
Shouldn't this be a method on Cholesky instead? (Probably the one that is `A`).

**Comment by kortschak** [comment]
`func (a *Cholesky) SolveTwo(m *Dense, b *Cholesky) error`? I don't know. It's sort of troubling since the destination is nestled in the middle.

**Comment by btracey** [comment]
Is it that different from `func (c *Cholesky) Solve(m *Dense, b Matrix) error`?

**Comment by kortschak** [comment]
Not really. I'll fix it.

**Comment by kortschak** [comment]
PTAL

File: mat/cholesky.go [hunk 1]
```diff
@@ -182,10 +182,9 @@ func (c *Cholesky) Solve(m *Dense, b Matrix) error {
 	return nil
 }
 
-// TODO(kortschak): Export this as SolveTwoChol.
-// solveTwoChol finds the matrix m that solves A * m = B where A and B are represented
+// SolveTwo finds the matrix m that solves A * m = B where A and B are represented
 // by their Cholesky decompositions a and b, placing the result in the receiver.
-func (m *Dense) solveTwoChol(a, b *Cholesky) error {
+func (a *Cholesky) SolveTwo(m *Dense, b *Cholesky) error {
```
Comment (btracey) [inline mat/cholesky.go]: Should this be `SolveChol` instead? Seems more evocative than `SolveTwo`, which means less now.

Comment (kortschak) [inline mat/cholesky.go]: Done

**Pull request merged**